src/components/icons/triangle/triangle.tsx

```diff
@@ -1,37 +1,18 @@
 import React from "react";
-import styled from "styled-components";
 
 const Triangle: React.FC<{ rotate: boolean }> = ({ rotate }) => {
-  const className = rotate ? "rotate" : "";
-
   return (
-    <Rotate>
-      <svg
-        width="11"
-        height="5"
-        viewBox="0 0 11 5"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-        className={className}
-      >
-        <path
-          d="M10.5 5L5.50003 5.96244e-08L0.500031 5L10.5 5Z"
-          fill="#03083C"
-        />
-      </svg>
-    </Rotate>
+    <svg
+      width="11"
+      height="5"
+      viewBox="0 0 11 5"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={rotate ? "mt-1 transform rotate-180" : ""}
```

Comment:
wasn't it a margin-bottom before?

Reply:
It was, but the value was `0.3rem`, a value not accessible with tailwind. So we changed it around so it was still align centered.